LN-Zap/zap-desktop
# refactor(deps): remove dependency on react-icons

## Description:

Remove dependency on react-icons

## Motivation and Context:

We only used react-icons in a couple of places, so it's overkill to pull in that dependency just for that. I've added the missing icons to our custom icon library and updated the code base to use our icons everywhere, so `react-icons` is no longer needed.

## How Has This Been Tested?

1. Navigate storybook and verify icons display correctly
2. Navigate app and verify icons display correctly 

## Types of changes:

Refactor

## Checklist:
